lib/mpl_toolkits/axes_grid1/axes_grid.py

```diff
@@ -22,23 +22,14 @@ def __init__(self, *args, orientation, **kwargs):
         self.orientation = orientation
         super().__init__(*args, **kwargs)
```

Comment:
Semi-OT, but got to this because you deleted `cla()`: This `super().__init__` should not do anything, right? - Likely a leftover from the dedicated axes_grid colorbar classes?

Reply:
It does (should?), as CbarAxesBase is a mixin and will thus end up in the base classes of the actual colorbar axes class used by ImageGrid, and thus we need to forward arguments via super().

Reply:
Indeed. Maybe that should be mentioned somewhere in the class? without that context, it's looks like `object` is the only parent. Either way, the PR is correct.